src/icons/general/IconBoxSquare.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    height?: number,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconBoxSquare = ({ className = '', height = 72, title, width = 72 }: Props) => (
```

Comment:
Instead of a new icon can we just use https://opensource.box.com/box-ui-elements/#/Components?id=logo with a CSS background?

Reply:
I've seen this particular icon around some Box UI (like the box favicon) that I think it would get re-use and should warrant a separate icon